### ISSUE TITLE
Enable “ButtonText” option for mutable array’s in config.schema.json

### ISSUE THREAD

**Issue by newmans99** [oznu/homebridge-config-ui-x#668]
In the config.schema.json file it would be helpful to support a Button Text entity to change the default “ADD TO {arrayname}” to something else.

For example...
"schema": {
"type": "object",
"properties": {
"calendars":{
"title": "Calendars",
“buttonText”:”Add a new calendar”,
"type": "array",
"orderable": false,
"required": true,
"description": "Add a calendar configuration.",
....

This would change the button from “ADD TO CALENDARS” to “Add a new calendar”.

Environment:

Node.js Version: All supported
NPM Version: All supported
Homebridge Version: 1.x+
Operating System: All supported
Process Supervisor: N/A

**Comment by oznu** [comment]
This is available for testing in 4.19.0-test.3.

https://github.com/oznu/homebridge-config-ui-x/wiki/How-To-Test-Upcoming-Changes

Example:

```diff
          "instanceBlacklist": {
            "title": "Blacklisted Instances",
+           "buttonText": "Add Instance",
            "type": "array",
            "items": {
              "title": "Instance Username",
              "type": "string",
            }
          }
```

**Comment by grzegorz914** [comment]
Tested, working fine.
